administration/src/util/translations/de.json

```
@@ -0,0 +1,72 @@
+    "street": "Straße",
+    "houseNumber": "Hausnummer",
+    "addressSupplement": "Adresszusatz",
+    "postalCode": "Postleitzahl",
+    "location": "Ort",
+    "country": "Land",
+    "dateOfBirth": "Geburtsdatum",
+    "emailAddress": "E-Mail-Adresse",
+    "region": "Zuständige Behörde",
+    "applicationDetails": "Antragsdetails",
+    "cardType": "Antrag auf",
+    "applicationType": "Art des Antrags",
+    "wantsDigitalCard": "Ich beantrage eine digitale Ehrenamtskarte",
+    "wantsPhysicalCard": "Ich beantrage eine physische Ehrenamtskarte",
+    "blueCardJuleicaEntitlement": "Ich bin Inhaber:in einer JuLeiCa (Jugendleiter:in-Card)",
+    "juleicaNumber": "Kartennummer",
+    "juleicaExpiration": "Karte gültig bis",
+    "copyOfJuleicaFront": "Kopie der Karte (1)",
+    "copyOfJuleicaBack": "Kopie der Karte (2)",
+    "blueCardWorkAtOrganizationsEntitlement": "Ich engagiere mich ehrenamtlich seit mindestens zwei Jahren freiwillig mindestens fünf Stunden pro Woche oder bei Projektarbeiten mindestens 250 Stunden jährlich",
+    "workAtOrganization": "Arbeit bei Organisation oder Verein",
+    "organization": {
+      "title": "Angaben zur Organisation",
+      "name": "Name der Organisation bzw. des Vereins",
+      "category": "Einsatzgebiet"
+    },
+    "organizationContact": {
+      "title": "Kontaktperson der Organisation",
+      "name": "Vor- und Nachname der Kontaktperson in der Organisation",
+      "telephone": "Telefon",
+      "email": "E-mail",
+      "hasGivenPermission": "Die Kontaktperson hat der Weitergabe ihrer Daten zum Zwecke der Antragsverarbeitung zugestimmt und darf zur Überprüfung kontaktiert werden"
+    },
+    "responsibility": "Tätigkeit",
+    "amountOfWork": "Arbeitsstunden pro Woche (Durchschnitt)",
+    "workSinceDate": "Tätig seit",
+    "payment": "Für diese ehrenamtliche Tätigkeit wurde eine Aufwandsentschädigung gewährt, die über den jährlichen Freibetrag hinaus geht (840 Euro Ehrenamtspauschale bzw. 3000 Euro Übungsleiterpauschale):",
+    "certificate": "Tätigkeitsnachweis",
+    "hasAcceptedPrivacyPolicy": "Ich erkläre mich damit einverstanden, dass meine Daten zum Zwecke der Antragsverarbeitung gespeichert werden und akzeptiere die Datenschutzerklärung",
+    "hasAcceptedEmailUsage": "Ich stimme zu, dass ich von der lokalen Ehrenamtskoordination über Verlosungen und regionale Angebote informiert werden darf",
+    "givenInformationIsCorrectAndComplete": "Ich versichere, dass alle angegebenen Informationen korrekt und vollständig sind",
+    "title": "Bayerische Ehrenamtskarte beantragen",
+    "sendSuccessfully": "Erfolgreich gesendet",
+    "sendError": "Fehler beim Senden",
+    "blueCardEntitlementType": {
+      "WorkAtOrganizations": "Ich engagiere mich ehrenamtlich seit mindestens zwei Jahren freiwillig mindestens fünf Stunden pro Woche oder bei Projektarbeiten mindestens 250 Stunden jährlich.",
+      "Juleica": "Ich bin Inhaber:in einer JuLeiCa (Jugendleiter:in-Card).",
```

:upside_down_face: I don't really like these nested namespaces and think that at least `EntitlementType` (and just `Entitlement`
 below) would not be needed in the keys. I guess this is personal preference though so feel free to ignore this.
```suggestion
    "blueCardWorkAtOrganizations": "Ich engagiere mich ehrenamtlich seit mindestens zwei Jahren freiwillig mindestens fünf Stunden pro Woche oder bei Projektarbeiten mindestens 250 Stunden jährlich.",
    "blueCardJuleica": "Ich bin Inhaber:in einer JuLeiCa (Jugendleiter:in-Card).",
```

---

Another note: I think in the code we mostly use `standard` and `gold` as card types, perhaps we could follow this here as well?

---

See explanation above why this is not possible.